packages/strapi-design-system/src/Button/Button.js

```diff
@@ -27,19 +27,33 @@ const BoxFullHeight = styled(Box)`
   height: 100%;
 `;
 
-// TODO: Check the L size button with Maeva
 export const ButtonWrapper = styled(BaseButton)`
   align-items: center;
-  padding: ${({ theme, size }) => `${size === 'S' ? theme.spaces[2] : '10px'} ${theme.spaces[4]}`};
-  background: ${({ theme }) => theme.colors.buttonPrimary600};
+  background-color: ${({ theme }) => theme.colors.buttonPrimary600};
   border: 1px solid ${({ theme }) => theme.colors.buttonPrimary600};
+  height: ${({ size }) => {
+    let height = 32;
+
+    if (size === 'M') {
```

Comment:
Can we use `BUTTON_SIZES`?

Reply:
I went for @udimberto s suggestion to move these into the theme constants in 875e6bc65b10f2beed1cac6deb09f1739922e415 . I'm not sure how valuable that is, but it feels good to have this in one place with the other constants. Does that work for you?

Reply:
yep 👍🏼